src/ft2font.h

```diff
@@ -18,8 +18,8 @@ extern "C" {
 /*
  By definition, FT_FIXED as 2 16bit values stored in a single long.
  */
-#define FIXED_MAJOR(val) (long)((val & 0xffff000) >> 16)
-#define FIXED_MINOR(val) (long)(val & 0xffff)
+#define FIXED_MAJOR(val) (signed short)((val & 0xffff0000) >> 16)
+#define FIXED_MINOR(val) (unsigned short)(val & 0xffff)
```

Comment:
Should also cause a type change in the version and fontrevision fields of converting TT_Header no?

Reply:
As in [here](https://github.com/matplotlib/matplotlib/pull/7853/commits/ea69e038d026c6272a7af0a309dde5a0606d12ae#diff-22fee04e99c432c6b01d4a2db592cad9)?